### ISSUE TITLE
Save recorded storage proof sizes in the database

### ISSUE THREAD

**Issue by jancionear** [near/nearcore#11102]
During chunk application we calculate how much storage proof was recorded for each receipt and chunk, along with the upper bounds. ([code](https://github.com/near/nearcore/blob/6d17fa939021e3853b916e5a54fd165b6bd2d348/runtime/runtime/src/lib.rs#L1444)).

These values are already exposed as metrics, but it'd be nice to also save them to the database. Metrics come and go along with the nodes, when a node is destroyed we lose continuity of its metrics.

Having these values in the archival databases would give us a solid source of data for analysis. We would be able to go over the past year and see how much storage proof was generated by receipts, which would allow us to confidently choose good values for the storage proof limits.

We could add it to the `ExecutionOutcome` struct, along with `gas_burnt`, `compute_usage`, etc.

Not necessary for stateless validation MVP, but would be nice to have.

**Comment by andrei-near** [comment]
How much storage will these proofs consume given current network load? Archival DB is without this already pretty large

**Comment by jancionear** [comment]
I'd like to save just the size of the proof, which would be a single `u64` per transaction/receipt.

**Comment by walnut-the-cat** [comment]
@jancionear , do you plan to take an action on it? Otherwise, let's close

**Comment by jancionear** [comment]
Yes I still plan to do it, this information could be included in `ChunkApplyStats`
